### DISCUSSION TITLE
Wallet address can not be found by Chiaststus / Chiaexplorer..

### DISCUSSION THREAD

**Discussion by firmicus** [Chia-Network/chia-blockchain#5292] [Support]
Dear All!
The wallet has 0 XCH for the time... but can not be found by Chiaststus / Chiaexplorer. 
On my host: 
- Current Blockchain Status: Full Node Synced 
- Ports are opened
- 10 to 15 FULL_NODE connections ...
(Got several versions updates ...)

What's going wrong ??

**Comment by firmicus** [comment]
I didn't put anything on blockchain so my wallet shouldn't be visible (known) to blockchain, right?

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
